src/brainBay.sln

```diff
@@ -1,19 +1,30 @@
-Microsoft Visual Studio Solution File, Format Version 11.00
-# Visual Studio 2010
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29009.5
+MinimumVisualStudioVersion = 10.0.40219.1
```

Comment:
I'm not sure if these changes might break compatibility with older versions of VS ...
are they necessary ?

Reply:
Hi Chris, I'm not sure. I think it was a log left by VS automatically. Given that it says minimum version to be 10 (seems to be VS 2010?), I've reverted these changes. If any issue encountered then we can still look into this. Does this sound ok to you? Thanks.